custom/conf/app.example.ini

```diff
@@ -1155,6 +1155,10 @@ LEVEL = Info
 ;;
 ;; Retarget child pull requests to the parent pull request branch target on merge of parent pull request. It only works on merged PRs where the head and base branch target the same repo.
 ;RETARGET_CHILDREN_ON_MERGE = true
+;;
+;; Delay mergeable check until page view or API access, for pull requests that have not been updated in the specified days when their base branches get updated.
+;; Use "-1" to always check all pull requests (old behavior). Use "0" to always delay the checks.
+;DELAY_CHECK_FOR_INACTIVE_DAYS = 7
```

Comment:
This seems like a good default to me.

Would this be considered an API breaking change? I guess you already have to poll in case it happens to be checking, and an API access will trigger the check to happen so it should eventually return an actual status.

Reply:
Hmm, maybe we could mention it in the changelog, and collect user feedbacks.